management_interface/management_interface/forms.py

```diff
@@ -50,12 +50,21 @@ def clean(self):
         self.cleaned_data["nhs_number"] = "".join(
             self.cleaned_data["nhs_number"].split()
         )
+        self.cleaned_data["nhs_number_hash"] = self._generate_nhs_number_hash()
+        already_existing_care_recipient = CareRecipient.objects.filter(
```

Comment:
Do we expect more than one? If not, we should probably use `objects.get()` to indicate we only query for a single object.

Reply:
This would raise `DoesNotExist` which is not a problem of course, we can catch it, but i wanted to prevent another level of nesting so the code is clearer to read

Reply:
Yes, definitely the error would be thrown. It was rather the "magic zero" index that makes me unhappy (like we ever wanted anything else than first element).